server/runtime/src/test/java/org/infinispan/server/functional/RestMetricsResource.java

```diff
@@ -35,10 +34,7 @@
 public class RestMetricsResource {
 
    @ClassRule
-   public static final InfinispanServerRule SERVERS =
-         InfinispanServerRuleBuilder.config("configuration/ClusteredServerTest.xml")
-                                    .numServers(2)
```

Comment:
My intention was to ensure we have at least 2 nodes running to ensure duplicate metric names in same registry, which then gets de-duplicated by the unique node tag.

Reply:
But the ClusteredIT is already running two nodes, and RestMetricsResource is being run as part of its suite

Reply:
oh, did not realize that :)